extension/llm/export/builder.py

```diff
@@ -354,15 +364,25 @@ def export_to_edge(self) -> "LLMEdgeManager":
             if self.pre_autograd_graph_module is None:
                 # Run export() if it didn't run
                 self.export()
-            self.edge_manager = export_to_edge(
-                self.pre_autograd_graph_module,  # pyre-fixme[6]
-                self.example_inputs,
-                example_kwarg_inputs=self.example_kwarg_inputs,
-                dynamic_shapes=dynamic_shape,
-                edge_constant_methods=self.metadata,
-                edge_compile_config=edge_config,
-                verbose=self.verbose,
-            )
+
+            override_export_behaviour = contextlib.nullcontext()
```

Comment:
Can you leave a TODO here and mention the issue you filed?

Reply:
ah - I left it at the top of the block.